src/Sarif/Sarif.csproj

```diff
@@ -19,7 +19,8 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461;net45</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;</TargetFrameworks>
+    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);net461;net45</TargetFrameworks>
```

Comment:
>OS [](start = 35, length = 2)

But wait. That means "If I'm _building_ on a non-Windows platform, then don't _build_ the .NET Framework version. Shouldn't we always build for the same set of targets, no matter _where_ we're building?

Reply:
when i was solving this issue, I couldn't find a way to install netframework on linux without using wine or other tools that simulate executable files from windows.

---
In reply to: [507937299](https://github.com/microsoft/sarif-sdk/pull/2122#discussion_r507937299) [](ancestors = 507937299)